rust/src/rfb/rfb.rs

```diff
@@ -206,8 +206,6 @@ impl RFBState {
 
                             if let Some(current_transaction) = self.get_current_tx() {
                                 current_transaction.ts_client_protocol_version = Some(request);
-                            } else {
-                                return AppLayerResult::err();
```

Comment:
When could this have happened ? (should it be a debug validation)

Reply:
I think it should. Will change all instances of this to a (much better suited) debug validation.

Reply:
Done.